docs/ftbbl.ipynb

```diff
@@ -0,0 +1,493 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ea00a41a",
+   "metadata": {},
+   "source": [
+    "# Feature Testbed Benchmark Library\n",
+    "\n",
+    "The Feature Testbed and Benchmarking Library (FTb/BL) consists of a series of workflow tools in `cirq_google.workflow` and a library of applications conforming to those specifications living in ReCirq.\n",
+    "\n",
+    "Computer code can serve as the intermediary language between algorithms researchers, software developers, and providers of quantum hardware -- each of whom have their own unique domain expertise. Higher-order abstraction in Quantum Executables can support higher-order functionality in our Quantum Runtime.\n",
+    "\n",
+    "By architecting features, tools, and techniques to be part of a well specified runtime we can:\n",
+    " - Easily run experiments (\"A/B testing\") on the impact of features by toggling them in the runtime and running against a full library of applications.\n",
+    " - Create a clear specification for how to leverage a feature, tool, or technique in our library of applications.\n",
+    " - Mock out runtime features for ease of testing without consuming the scarce resource of quantum computer time."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b1fcbd4",
+   "metadata": {},
+   "source": [
+    "## Algorithmic Benchmark Library\n",
+    "\n",
+    "The algorithmic benchmark library is a collection of `QuantumExecutableGroup`s that probe different aspects of a quantum computer's performance. Each benchmark is based off of an algorithm of interest and operates on > 2 qubits (in contrast to traditional 1- and 2-qubit fidelity metrics).\n",
+    "\n",
+    "We use the library card catalog in `recirq.algorithmic_benchmark_library` to get a description of each algorithmic benchmark."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dfa69118",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import recirq.algorithmic_benchmark_library as algos\n",
+    "\n",
+    "for algo in algos.BENCHMARKS:\n",
+    "    display(algo)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "72765c1f",
+   "metadata": {},
+   "source": [
+    "### Unique `executable_family`\n",
+    "\n",
+    "Each benchmark has a globally-unique `executable_family` string, as well as other helpful identifiers. For example: the `recirq.otoc.loschmidt.tilted_square_lattice` has the domain of `recirq.otoc`, as it is similar to circuits run as part of the [OTOC Experiment](https://quantumai.google/cirq/experiments/otoc). We'll select this one for further inspection."
```

Comment:
It feels like this is missing a sentence that explains what an `executable_family` string actually is (and perhaps its scope, like how broad or narrow is a family).  Or maybe move the below "In ReCirq, the executable_family is the same as the Python module path" up here instead?